src/lang/Messages.properties

```diff
@@ -439,8 +439,14 @@ authorization.api.view.getAuthorizationDetectionMethod		= Obtains all the config
 authorization.api.action.setBasicAuthorizationDetectionMethod	= Sets the authorization detection method for a context as one that identifies un-authorized messages based on: the message's status code or a regex pattern in the response's header or body. Also, whether all conditions must match or just some can be specified via the logicalOperator parameter, which accepts two values: "AND" (default), "OR".  
 
 autoupdate.api.action.downloadLatestRelease = Downloads the latest release, if any 
+autoupdate.api.action.installAddon = Installs the specified add-on, returning when complete (ie not asynchronously)
+autoupdate.api.action.uninstallAddon = Uninstalls the specified add-on 
 autoupdate.api.view.latestVersionNumber = Returns the latest version number
 autoupdate.api.view.isLatestVersion = Returns 'true' if ZAP is on the latest version
+autoupdate.api.view.installedAddons = Return a list of all of the installed add-ons
+autoupdate.api.view.newAddons = Return a list of any add-ons that have been added since the last check for updates
```

Comment:
I think you should be more specific here, I assume you mean added to the marketplace (not to the client installation).